easybuild/easyconfigs/c/COMSOL/COMSOL-5.3a.eb

```diff
@@ -0,0 +1,14 @@
+name = 'COMSOL'
+version = '5.3a'
+
+homepage = 'https://www.comsol.se/'
+description = """COMSOL Multiphysics - Simulation Tool for Electrical, Mechanical, Fluid Flow, and Chemical
+Applications"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = [SOURCELOWER_TAR_GZ]
```

Comment:
missing checksum (also, a comment on where to find this file?)

Reply:
The file has to be created manually from the `COMSOL53a_dvd.iso` (so including a checksum is useless).

But, if we fix https://github.com/easybuilders/easybuild-framework/issues/1376, we can consider using the ISO file as a source, and unpacking that instead?

Reply:
I've updated the PR to use `None` as checksum, which is a supported escape hatch that is acceptable in this case imho...